platform/spec/services/bloom_filter_service_spec.rb

```
@@ -0,0 +1,130 @@
+
+  before do
+    redis.del(test_key) if redis.exists?(test_key)
+  rescue Redis::CommandError
+    # Handle missing Redis Bloom module
+  end
+
+  after do
+    redis.del(test_key) # Clean up after each test
+  end
+
+  before(:all) do
```

RuboCop (1.73)

[convention] Beware of using `before(:all)` as it may cause state to leak between tests. If you are using `rspec-rails`, and `use_transactional_fixtures` is enabled, then records created in `before(:all)` are not automatically rolled back.

(RSpec/BeforeAfterAll)